lib/sidekiq_alive/worker.rb

```diff
@@ -3,17 +3,23 @@
 module SidekiqAlive
   class Worker
     include Sidekiq::Worker
-    sidekiq_options retry: false
+    sidekiq_options retry: true
 
     # Passing the hostname argument it's only for debugging enqueued jobs
     def perform(_hostname = SidekiqAlive.hostname)
       # Checks if custom liveness probe passes should fail or return false
-      return unless config.custom_liveness_probe.call
+      unless config.custom_liveness_probe.call
```

Comment:
Since you did not override the custom_liveness_probe config [here](https://github.com/AcadiaMicro/lebowski/pull/4895/files#diff-d9b440751dafde4afe315a91d797b4c12fee2a03f33431c0181cbe1702b87064), the default just returns 'true', so what is the purpose of this logging message?  Would it have been better to add the log message to the rescue in SidekiqAlive::Worker#write_living_probe?

Reply:
My understanding is that it will return `false` if the probe fails. Is that not correct?

Reply:
Yeah, that's a problem. I'll remove it if/until we override the liveness probe.